.github/workflows/docker.yaml

```diff
@@ -24,15 +24,37 @@ jobs:
       uses: docker/setup-buildx-action@v1
 
     - name: Login
+      if: ${{ github.event_name != 'pull_request' }}
       uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
-        
+
+    - name: Login to GitHub Container Registry
+      if: ${{ github.event_name != 'pull_request' }}
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
     - name: Build and push
       uses: docker/build-push-action@v2
       with:
         context: .
-        push: true
-        tags: mpadge/pkgcheck:latest
-        
+        push: ${{ github.event_name != 'pull_request' }}
+        tags: |
+          mpadge/pkgcheck:latest
+          ghcr.io/ropensci-review-tools/pkgcheck:latest
+
+    - name: Trigger pkgcheck-action build
+      if: ${{ github.event_name != 'pull_request' }}
+      uses: actions/github-script@v5
+      with:
+        script: |
+          await github.rest.actions.createWorkflowDispatch({
+            'ropensci-review-tools',
+            'pkgcheck-action',
+            'publish.yaml',
```

Comment:
@assignUser Just so i understand everything here - what is this "publish.yaml" line?

Reply:
This will trigger the [build action](https://github.com/assignUser/pkgcheck-action/blob/1ab47d1574a83d5ef2a1b6ebad32d2d45cf019b8/.github/workflows/publish.yaml) in the pkgcheck-action  repo, so anytime this container is updated the pkgchk-action container is also rebuild.